lib/testers/rule-tester.js

```diff
@@ -553,7 +557,7 @@ class RuleTester {
                     } else {
 
                         // Message was an unexpected type
-                        assert.fail(message, null, "Error should be a string, object, or RegExp.");
+                        assert.fail("Error should be a string, object, or RegExp.");
```

Comment:
What versions of Node support `assert.fail` with one argument? Currently, we're supporting Node 4 on master (although we are dropping support for Node 4 and 5 on master in a few days).

Reply:
Oh, I forgot about that again. You are right, I'll revert that change.